airflow/jobs/scheduler_job_runner.py

```diff
@@ -1712,7 +1725,10 @@ def _find_zombies(self) -> None:
                 simple_task_instance=SimpleTaskInstance.from_ti(ti),
                 msg=str(zombie_message_details),
             )
-            self.log.error("Detected zombie job: %s", request)
+            # Refer: https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#zombie-undead-tasks
+            log_message = f"Detected zombie job: {request}"
+            self.log.error(log_message)
```

Comment:
We may not need this line as the task_context_logger logs it with the self.log object we pass to it. Thus avoiding the need to have multiple log.error statements

Reply:
Thanks. Just removed it